### ISSUE TITLE
Please Tell Me The Runtime Environment

### ISSUE THREAD

**Issue by velopt** [blaufish/Tpm2AttestationCertificationAuthority#1]
I'm working on the PrivacyCA, and found this project. It's a great experiment, easy to understand and is already perfect.

So I want to run this project first, to understand the whole process. The tpm2-tools part is clear to me. The following is the Java part that I want to confirm:

Java: JDK 15, I use openjdk-15.0.2
External jar: bcpkix-jdk15on-1.60.jar, bcprov-jdk15on-1.60.jar
For OpenCit, I think the 3 code files attached in project are enough. Later I'll study on the whole OpenCit.

If there is anything wrong or something I didn't mentioned, PLEASE tell me.

By the way, is the tpm manufacteror cert downloaded from TCG website? I think I have seen it before.

**Comment by blaufish** [comment]
Hi! **Java part** 
- Yes, you seem to reference same files as the **README.md** references. 
- You also have eclipse .project and .classpath to give some further hints to what my project setup was back in the day. 

**is the tpm manufacteror cert downloaded from TCG website?**
- Not 100% certain if I got it from the TCG website or from the manufacturer certificate.
- You can find Infineon TPM manufacturer certificates at URLs like this: http://pki.infineon.com/OptigaRsaMfrCA022/OptigaRsaMfrCA022.crt
- If you have your TPM certificate, you can usually locate the manufacturer certificate based on certificate **Authority Information Access** extension and then (based on some moderately secure administrative process) considering adding it to your trust anchors / key store / ... system.
- It was the certificate corresponding to the discreet TPM & hardware we were proof of conception stuff against back in the day.

Hope this was helpful, I haven't worked this project in 5 years time.

**Comment by velopt** [comment]
Thank you for your reply. Everything I predicted was just right. My TPM was provided by Intel CPU, which is called INTC. I downloaded manufacteror cert from the EK cert, ISSUSER part precisely. And the whole project worked on Thursday, two days ago.
I also completed the VerifierCli. And got a "true".
Next I'll try to put this into application, network accessing for example.
In future, maybe I'll test the code on Infineon machine, then your reply will be very useful.

Anyway, your project and your help to me means a lot! Thank you.
